galeracluster/source/documentation/mysql-wsrep-options.rst

```diff
@@ -1005,19 +1005,39 @@ Extends node behaviour with provided values.
    "Command-line Format", "``--wsrep_mode``"
    "System Variable", "``wsrep_mode``"
    "Variable Scope", "Global"
-   "Dynamic Variable", ""
+   "Dynamic Variable", "Yes"
    "Permitted Values", "Set"
-   "Default Value", "APPLIER_IGNORE_MISSING_TABLE"
+   "Default Value", "See the information below."
    "Initial Version", "MySQL-wsrep: 5.7.32-25.24, 8.0.22-26.5, MariaDB: 10.6.0"
 
 .. csv-table::
    :class: doc-options
-   :header: "Value", "Behaviour"
-   :widths: 25, 50
+   :header: "Galera Cluster version", "MySQL database server version", "Blog post"
+
+The options for MySQL are:
+
+- ``IGNORE_NATIVE_REPLICATION_FILTER_RULES`` - Ignore native replication filter rules for cluster events. In other words, native asynchronous replication filtering options are honored when applying Galera replication. These options are of format ``replicate_*``, and specify if transactions for a table or a database should be applied or not.
+- ``IGNORE_CASCADING_FK_DELETE_MISSING_ROW_ERROR`` - Ignore missing row errors when applying a cascading delete write set. This a workaround for https://bugs.mysql.com/bug.php?id=80821, and is possibly obsolete in the upstream versions.
```

Comment:
can you make this an actual link?

Reply:
In the conversion that I am looking at, it already is a link. Does it not work on your conversion?

Reply:
Hmm, ok, i think it does now. Weird